tools/src/versioning/ios/versionVendoredModules.ts

```diff
@@ -122,6 +122,10 @@ function baseTransformsFactory(prefix: string): Required<FileTransforms> {
         find: /react::/g,
         replaceWith: `${prefix}React::`,
       },
+      {
+        find: /hermes::/g,
+        replaceWith: `${prefix}hermes::`,
```

Comment:
What would you say if we merge this and two previous rules into:
```suggestion
        find: /(facebook|react|hermes)::/g,
        replaceWith: (_, p1) => {
          return `${prefix}${p1 === 'react' ? 'React' : p1}::`;
        },
```